guide/additional-info/changes-in-v12.md

```diff
@@ -14,6 +14,34 @@ For now, you do need Git installed and added to your PATH environment, so ensure
 * The use of parenthesis designates optional inclusion. For example, `Channel#fetch(Pinned)Message(s)` means that this section will include changes for `Channel#fetchPinnedMessages`, `Channel#fetchMessages`, and `Channel#fetchMessage`.
 * The use of asterisks designates a wildcard. For example, `Channel#send***` means that this section will include changes for `Channel#sendMessage`, `Channel#sendFile`, `Channel#sendEmbed`, and so forth.
 
+### Collection
+
+#### Collection#exists
+
+`collection.exists` was removed entirely, `collection.some()` should be used to check if an element exists in the collection that satisfies the provided value.
```

Comment:
To stay consistent with the other examples `collection.exists` should be `collection.exists()`

Reply:
Fixed.